AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/README.md

```diff
@@ -0,0 +1,83 @@
+# Tutorial : Optimize TensorFlow pre-trained model for inference
+This tutorial will guide you how to optimize a pre-trained model for a better inference performance, and also 
+analyze the model pb files before and after the inference optimizations.  
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Ubuntu* 18.04 
+| Hardware                          | Intel® Xeon® Scalable processor family or newer
+| Software                          | [Intel® oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html)
+| What you will learn               | Optimize a pre-trained model for a better inference performance
+| Time to complete                  | 30 minutes
+
+## Purpose
+Show users the importance of inference optimization on performance, and also analyze TensorFlow ops difference in pre-trained models before/after the optimizations.  
+Those optimizations includes:  
```

Comment:
Those optimizations **includes** --> Those optimizations **include**

Reply:
addressed it accordingly.